Jenkinsfile

```diff
@@ -27,37 +27,40 @@ if (needSplittingFromWorkspace) {
 for (int i = 0; i < splits.size(); i++) {
     int index = i
     for (int j in [11]) {
-        int javaVersion = j
-        def name = "java-${javaVersion}-split${index}"
-        branches[name] = {
-            stage(name) {
-                node('docker-highmem') {
-                    checkout scm
-                    def image = docker.build('jenkins/ath', '--build-arg uid="$(id -u)" --build-arg gid="$(id -g)" ./src/main/resources/ath-container/')
-                    image.inside('-v /var/run/docker.sock:/var/run/docker.sock --shm-size 2g') {
-                        def exclusions = splits.get(index).join("\n")
-                        writeFile file: 'excludes.txt', text: exclusions
-                        realtimeJUnit(
-                                testResults: 'target/surefire-reports/TEST-*.xml',
-                                testDataPublishers: [[$class: 'AttachmentPublisher']],
-                                // Slow test(s) removal can causes a split to get empty which otherwise fails the build.
-                                // The build failure prevents parallel tests executor to realize the tests are gone so same
-                                // split is run to execute and report zero tests - which fails the build. Permit the test
-                                // results to be empty to break the circle: build after removal executes one empty split
-                                // but not letting the build to fail will cause next build not to try those tests again.
-                                allowEmptyResults: true
-                        ) {
-                            sh """
-                                set-java.sh $javaVersion
-                                eval \$(vnc.sh)
-                                java -version
+      for (String v in ["lts", "latest"]) {
+          int javaVersion = j
+          String jenkinsUnderTest = v
```

Comment:
what's the point to this? just name it jenkins under test in the loop same with java version?

Reply:
https://www.jenkins.io/doc/pipeline/examples/#jobs-in-parallel